packages/SwingSet/src/controller/controller.js

```diff
@@ -562,7 +551,7 @@ export async function buildVatController(
     debugPrefix,
     slogCallbacks,
     warehousePolicy,
-    slogFile,
+    slogSender,
```

Comment:
I see why `slogFile` had to be removed, but I gotta admit that's going to interfere with some debugging I frequently do within swingset unit tests, where it's awfully handy to add a one-line change to the `buildSwingsetController` call to record the test actions. I guess I'll write a helper function in `test/` to make that trick possible again, the next time I need it.

Reply:
It should be pretty easy to write a `slogSender` for tests that need to record slog events. It'd be a better abstraction anyway than relying on an intermediate slog file. Probably something along the lines of `test-create-vat.js`'s `slogSender`.